feature/onboarding/success/src/main/kotlin/app/k9mail/feature/onboarding/success/ui/SuccessContent.kt

```diff
@@ -0,0 +1,91 @@
+package app.k9mail.feature.onboarding.success.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import app.k9mail.core.ui.compose.designsystem.atom.button.Button
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextBody1
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextHeadline4
+import app.k9mail.core.ui.compose.theme.MainTheme
+import app.k9mail.feature.onboarding.success.R
+
+@Composable
+internal fun SuccessLogo(
```

Comment:
Instead of writing composables inside `SuccessContent`, It should come under `components` package (used for defining reusable composables). What do you think?